ui/index.d.ts

```diff
@@ -360,6 +362,128 @@ export declare abstract class AzureWizardStep<T> {
     public abstract execute(wizardContext: T, outputChannel: OutputChannel): Promise<T>;
 }
 
+export interface ISubscriptionWizardContext {
+    credentials: ServiceClientCredentials;
+    subscription: Subscription;
+}
+
+export interface ILocationWizardContext extends ISubscriptionWizardContext {
+    /**
+     * You may specify the defaultLocationName if you don't want the `LocationStep` to prompt for a location
+     * For example, if the user selects an existing resource, you might want to use that location as the default for the wizard's other resources
+     */
+    defaultLocationName?: string;
```

Comment:
This is confusing - default implies you're going to ask the user but ENTER will select this. Would it make sense to just set location instead?  Or maybe defaultLocationNoPrompt?

Reply:
I see what you mean. Refactored this so that you call `LocationStep.setLocation` and removed the defaultLocationName property